src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.html

```diff
@@ -1,6 +1,7 @@
 <button type="button" class="btn btn-sm btn--capture-keystroke"
             [ngClass]="{'btn-default': !record, 'btn-info': record}"
-            (click)="start()"
+            (keydown.enter)="onButtonEnterDown($event)"
```

Comment:
There were also problems with the interception, with the recording the popover also closed.
(It was propagated however it shouldn't be when it was recorded.).
My solution fixes those and this was easiest solution for me to fix those bugs.